contracts/infrastructure/storage/ILockStorage.sol

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.5.4 <0.7.0;
+
+interface ILockStorage {
+    function isLocked(address _wallet) external view returns (bool);
+
+    function getLock(address _wallet) external view returns (uint256);
+
+    function getLocker(address _wallet) external view returns (address);
+
+    function setLock(address _wallet, address _locker, uint256 _releaseAfter) external;
```

Comment:
If the above will not be used on the GuardianStorage, can we remove them from its interface? I think all modules only go via the interface rather than the implementation contract.

Reply:
Unfortunately, for some reason, removing these methods from `IGuardianStorage` changes the deployed bytecode of `GuardianStorage`, which we probably don't want to do since `GuardianStorage` is not redeployed.